docs/partial_source/deep_dive/ivy_frontends_tests.rst

```diff
@@ -490,6 +490,74 @@ This function requires us to create extra functions for generating :code:`shape`
 * We use :code:`helpers.get_dtypes` to generate :code:`dtype`, these are valid numeric data types specifically for Torch.
 * :func:`torch.full` supports :code:`out` so we generate :code:`with_out`.
 
+Testing Without Using Tests Values
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are some places where we cannot test using hypothesis, as some functions have multiple solutions and all those
```

Comment:
We actually still use Hypothesis, it's just how our definition of `Equality` is different, for example we can still have Hypothesis to generate random values for `x`, if we have a function in our backend that is called `add_noise` and we call it on `x` and on we try to assert (we interally use `assert np.all_close`) that the result from `torch` backend matches `tensorflow` the test will always fail, because the function `add_noise` depends on a random seed internally that we have no control over, what we change is only how we test for equality, in which in that case we can not.